packages/internal-scripts/src/add-browser-exports.ts

```diff
@@ -0,0 +1,213 @@
+/**
```

Comment:
I'd like if this could be run as a test script as part of `npm run test`. The test wouldn't ever write anything to the filesystem, it'd just print something and exit with a nonzero exit code if this script would normally have written something (and mention that you can run the script to fix the failure).

It's a lightweight way to be sure we maintain this invariant in our package.json files

Reply:
Excellent comment! I can definitely add that!

Reply:
Done, I've set this up to run as part of the top level `test` script.

Verified by manually adding an error that it will signal a failure. Attached screenshot:

<img width="670" alt="Screenshot 2024-01-10 at 2 31 06 PM" src="https://github.com/lit/lit/assets/15080861/d575f7c3-d053-44b6-9c34-edb3d290d4e5">